edge/docs/installation/inception_internals.md

```diff
@@ -0,0 +1,122 @@
+# IBM Maximo Visual Inspection Edge Inception Internals
+
+Inception is the process by which MVIE is installed. In the most simple deployments this process reduces user input to 2 commands but for a typical production deployment several other steps will be required. There are also a number of non-default configurations that cannot be changed from the UI. This document presents the basic inception process and additional changes that might be desired in different scenarios.
+
+### Inception Basics
+In the simplest case, MVIE is installed as follows:
+1. The [Installation pre-requisites](https://www.ibm.com/docs/en/maximo-vi/8.3.0?topic=edge-planning) are provided
```

Comment:
Do you want a specific version reference in the link? 
Does this imply that you will have to update this file regularly?

Reply:
> Do you want a specific version reference in the link?
> Does this imply that you will have to update this file regularly?

I don't know of a better way to link to the documentation and yes, this will have to be updated whenever we change the pre-requisites.